src/domain-services/location/location-service.ts

```diff
@@ -0,0 +1,23 @@
+import { Service } from 'typedi';
+import { DbModels } from '../../data-providers/postgres/models';
+import { LocationTable } from '../../data-providers/postgres/models/location';
+
+@Service()
+export class LocationService {
```

Comment:
As we're moving back to `typegraphql`'s way of doing things, it is worth updating the file structure doc https://github.com/UN-OCHA/hpc-api/blob/develop/docs/FILE_STRUCTURE.md.

We previously were going to call our db abstractions libs, however `typedi` follows the spring structure of calling these services.

Reply:
Did some changes to mention services are used with help of dependency injection from `typedi` - see ba800a8aeb877bac6f4685d85793dadffd398d62.